Assets/Scripts/Level Scripts/FirstLevel.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using UnityEngine;
+
+
+// This script is for controlling the tutorials in the first level
+// Preferably this would've been made dynamically with scriptable objects and such
+// However, due to time limitations, This will be done the ugly but fast way.
+
+public class FirstLevel : MonoBehaviour
+{
+    [SerializeField]
+    private TutorialUIController _tutorialUi;
```

Comment:
_tutorialUi 
not _tutorialUI due to UI being an abbreviation?

Reply:
Fixed